modules/lnd/lnd_wrapper/wrapper.go

```diff
@@ -162,6 +162,31 @@ func LndParseP2pLightningMessage(data *C.char, length C.int) *C.char {
 	case 19:
 		sb.WriteString("MSG_TYPE=pong;IGNORED=")
 		sb.WriteString(fmt.Sprintf("%d", len(message.(*lnwire.Pong).PongBytes)))
+	case 34:
+		_, err := message.(*lnwire.FundingCreated).CommitSig.ToSignature()
+		if err != nil {
+			// Both rust-lightning and c-lightning verify the signature only
+			// when verifying against the public key.
```

Comment:
This comment is confusing to me.  Is it trying to say that LND does some validation that LDK and CLN don't?